### PR TITLE
block_scsi_device: New a case to test scsi device

### PR DESCRIPTION
**Pull request by YongxueHong**
New a case to test scsi device by deleting and scanning.

ID: 1731608
Signed-off-by: Yongxue Hong <yhong@redhat.com>

### CODE REVIEW

File: qemu/tests/block_scsi_device.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        """ Delete the scsi drive. """
+        error_context.context(
+            "Delete the scsi driver @%s." % scsi_addr, logging.info)
+        session.cmd('echo 1 > /sys/class/scsi_device/%s/device/delete' % scsi_addr)
+
+    def get_scsi_addr_by_product(product_name):
+        """ Get the scsi address by virtio_scsi product option. """
+        logging.info("Get the scsi address by qemu product option.")
+        addr = session.cmd(
+            "lsscsi | grep %s | awk '{print $1}'" % product_name).strip()[1:-1]
```
Comment (lolyu) [inline qemu/tests/block_scsi_device.py]: could use `strip("[]")`

Comment (YongxueHong) [reply qemu/tests/block_scsi_device.py]: ok, thanks, have updated.

File: qemu/tests/block_scsi_device.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        session.cmd('echo 1 > /sys/class/scsi_device/%s/device/delete' % scsi_addr)
+
+    def get_scsi_addr_by_product(product_name):
+        """ Get the scsi address by virtio_scsi product option. """
+        logging.info("Get the scsi address by qemu product option.")
+        addr = session.cmd(
+            "lsscsi | grep %s | awk '{print $1}'" % product_name).strip('[] ')
+        logging.info(
+            "The scsi address of the product %s is %s." % (product_name, addr))
+        return addr
```
Comment (XueqiangWei) [inline qemu/tests/block_scsi_device.py]: Failed to get addr. logs:

5:53:55 INFO | The scsi address of the product "AUTO HARDDISK" is 6:0:1:0]
.
05:53:55 INFO | Context: Delete the scsi driver @6:0:1:0]
.
05:53:55 DEBUG| Sending command: echo 1 > /sys/class/scsi_device/6:0:1:0]
/device/delete

Comment (YongxueHong) [reply qemu/tests/block_scsi_device.py]: @XueqiangWei , thanks,  have updated it.

**Comment by XueqiangWei** [comment]
(1/1) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.7.x86_64.io-github-autotest-qemu.block_scsi_device.from_delete_to_scan.q35: PASS (75.15 s)

LGTM,
Acked-by xuwei@redhat.com

**Pull request merged**